src/applications/personalization/profile-2/components/direct-deposit/DirectDepositContent.jsx

```diff
@@ -249,14 +249,15 @@ export const DirectDepositContent = ({
       value: (
         <div className="vads-u-display--flex vads-u-flex-direction--column">
           <p className="vads-u-margin-top--0">
-            You’ll need to sign in to the eBenefits website with your Premium DS
-            Logon account to change your direct deposit information for GI Bill
-            and other education benefits online.
+            You’ll need to sign in to the eBenefits website with your{' '}
+            <b>Premium DS Logon</b> account to change your direct deposit
```

Comment:
I remember learning years ago that `<strong>` is preferred to `<b>` but really am not sure. I wonder if @jenstrickland has an opinion on this since she's the semantic markup maven

Reply:
"The difference between the two HTML tags is that bold makes text only visually look bold, while strong also semantically emphasizes the respective text as important and indicates that it is a meaningful word or text section." - from https://www.seobility.net/en/wiki/Strong_and_Bold_Tags

I can do either depending on the preference.

Reply:
Yes, `<strong>` is preferred, as it will communicate the emphasis. I haven't heard a difference — yet — in screen readers, but that's how I was trained with the semantic old skool kids.